src/AstroBase.jl

```diff
@@ -1,5 +1,23 @@
 module AstroBase
 
-# package code goes here
+using Rotations
+export polar_motion
 
-end # module
+"""
+    polar_motion(rx, ry, sp)
+
+Form the matrix of polar motion for coordinates of the pole (radians).
+```
+jldoctest
+julia> polar_motion(20, 30, 50)
+3×3 RotZYX{Int64}(50, -20, -30):
```

Comment:
s/Int64/Float64/

Reply:
Ah, no sorry. This is Unix jargon for `replace Int64 with Float64`. It's the syntax that tools like vim and sed use for this kind of task.